webapp/src/utils/lyraConfig.ts

```diff
@@ -67,7 +64,6 @@ export class LyraConfig {
             project.languages ?? ['en'], // default language to be english if not provided
           );
         }),
-        parsed.baseBranch ?? 'main', // default base branch to be main if not provided
```

Comment:
This was the last usage of baseBranch of lyraConfigSchema. Let's remove it there too.

While I suspect it will break compatibility with existing configuration files, 1) that does not hurt so much as it only affects me, you and Rickard, 2) an error message from the parser would actually help because it shows that we have to update our configuration files.

Reply:
fixed, it will throws LyraConfigReadingError if baseBranch defined in lyraConfig.
currently we are not logging error messages due to some error with pipeline